lms/static/js/certificates/views/certificate_bulk_whitelist.js

```diff
@@ -25,7 +25,8 @@
                 data_format_error: 'data-format-error',
                 user_not_exist: 'user-not-exist',
                 user_already_white_listed: 'user-already-white-listed',
-                user_not_enrolled: 'user-not-enrolled'
+                user_not_enrolled: 'user-not-enrolled',
```

Comment:
General note: Changes in this file look more extensive than they really are. I will try to point out the parts that I added or changed. There were a number of JS quality/linting errors that I had to fix because I think it's been awhile since this file had been modified.